src/components/complete-profile/CompleteProfileBlock.tsx

```diff
@@ -62,16 +61,19 @@ const CompleteProfileBlock: FC<CompleteProfileBlockProps> = ({
     [userId]
   )
 
-  const { response, fetchData } = useAxios({
-    service: getMyOffers,
-    defaultResponse
+  const { data: queryData, refetch } = useQuery({
+    queryKey: ['complete-profile-block', userId],
+    queryFn: async () => {
+      const response = await getMyOffers()
+      return response.data as { items: Offer[]; count: number }
+    }
   })
 
   useEffect(() => {
```

Comment:
You should not fetch data in `useEffect` if you are using `useQuery` hook.

Reply:
Ok, so should i add isOfferCreated to queryKey?

Reply:
Fixed.